Oxide.Core/Plugins/Watchers/FSWatcher.cs

```diff
@@ -53,14 +53,14 @@ private void LoadWatcher(string directory, string filter)
             // Create the watcher
             watcher = new FileSystemWatcher(directory, filter)
             {
-                EnableRaisingEvents = true,
                 IncludeSubdirectories = true,
```

Comment:
Might as well move the rest of these to the same format as the others at this point. :P

Reply:
Gimme a sec. :)